src/gc/impl/conservative/gc.d

```diff
@@ -3009,6 +3152,7 @@ struct Pool
             freebits.alloc(nbits);
             freebits.setRange(0, nbits);
         }
+        freebits.setAll();
```

Comment:
This is already done with the `freebits.setRange(0, nbits);` call above.

Reply:
I get a sigsegv if I replace that call with  freebits.setRange(0, nbits);
setAll does this: `memset(data, 0xFF, nwords * wordtype.sizeof) `

Reply:
freebits is only allocated for SmallObjectPool, but setAll is called unconditionally. I suspect setRange doesn't work as expected with nbits == 0.

Reply:
Remove this line or replace the freebits.setRange call above with it.

Reply:
Why do you want me to remove that line? 
It leaks without it while setRange generates a sigsegv because it does a different thing.
I can put it behind an else statement so that it gets called for smallPools only.

Reply:
freebits is empty in the else case, so it does the same as setRange for small object pools.